iFixit/react-commerce
# Improve badge component

closes #687 

This PR updates the base chakra `Badge` component styles, to reflect the badge as designed in the mockup.

## QA

1. Visit Vercel preview
2. Visit `/Parts` product list
3. Verify that product list badges appear as in the mockup

| Before  | After |
| ------------- | ------------- |
| <img width="169" alt="Screenshot 2022-09-21 at 14 25 47" src="https://user-images.githubusercontent.com/4640135/191503154-c3830b52-b64c-4d35-b040-16b2ea513a8c.png">  |  ![Screenshot 2022-09-21 at 14 26 12](https://user-images.githubusercontent.com/4640135/191503241-98a76f85-39a7-40d1-857b-406831d34428.png) |

4. Verify that the grid view also has the new badge (the current version is not showing badges, in this new updated version we'll show just one badge

There are badges also in the product page, but most of them are not visible right now because metafields have been deleted from the test store. Anyway I think we can merge this PR if everything looks fine for the product list pages 👍
